gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,132 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord, Angle
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    @property
+    def offset(self):
+        """
+        Compute offsets of the different observations
+        
+        Returns
+        -------
+        vector : `~astropy.Angle`
+            Offsets
+        """
+        pnt_pos = SkyCoord(self.obs_table['RA_PNT'],
+                           self.obs_table['DEC_PNT'],
+                           unit='deg')
+
+        offset = pnt_pos.separation(self.target_pos)
+        
+        return offset.degree
+    
+    def plot_zenith_distribution(self, ax=None, bins=None):
+        """
+        Construct the zenith distribution of the observations
+        
+        Parameters
+        ----------
+        ax : `~matplolib.axes`, optional
+            Axis
+        bins : integer
+            number of bins, optional
+        range : `range`
```

Comment:
The docstring now no longer matches the function signature.
Remove the `range` option and update the description of `bins`.